recipe/meta.yaml

```diff
@@ -22,7 +21,7 @@ requirements:
     - wheel
     - setuptools
   run:
-    - python >=3.6
+    - python   
```

Comment:
There might be some whitespace here at the end of this line (after `python`). Doesn't seem to be a problem with this build (because all of them were successful) but whitespace has been known to cause other headaches so may be best to just remove it. Thank you! 🙏

Reply:
Will do! thanks for the explanation of this!